lib/Runtime/Library/InJavascript/Intl.js

```diff
@@ -827,7 +848,7 @@
         // using const f = function ... to remain consistent with the rest of the file,
         // but the following function expressions get a name themselves to satisfy Intl.Collator.name
         // and Intl.Collator.prototype.compare.name
-        const Collator = tagPublicFunction("Intl.Collator", function Collator(locales, options) {
+        const Collator = tagPublicFunction("Intl.Collator", function Collator(locales = undefined, options = undefined) {
```

Comment:
>locales = undefined, options = undefined [](start = 78, length = 40)

If i'm not mistaken this should change Intl.Collator.length from 2 to 0 but it looks like it is already 0 in latest master (and all other engines agree on 0).

Reply:
Yeah, I saw that pattern before and said "that seems silly, why are we doing that?" and then test262 reminded me of why

Reply:
We should add comments about this (every place we do this)

---
In reply to: [159295947](https://github.com/Microsoft/ChakraCore/pull/4478#discussion_r159295947) [](ancestors = 159295947)